content/docs/0.8.x/operate/multi_cluster/index.md

```diff
@@ -0,0 +1,220 @@
+  
+  * Project: `sockshop`
+  * Service: `carts`
+  * Stages: 
+      * `hardening` - on Cluster-A
+      * `production` - on Cluster-B  
+
+    To properly configure the execution plane services that run, for example, on **Cluster-A**, the distributor in the `values.yaml` needs to be configured:
+
+    ```
+    distributor:
+      projectFilter: ""                     # set the project, e.g., "sockshop" (to get events for the entire project)
+      stageFilter: ""                       # set the stage, e.g., "hardening" (to get events for the stage)
+      serviceFilter: ""                     # set the service, e.g., "carts" (to get events for the service )
+    ``` 
+
+* Deploy the execution plane service (e.g., jmeter-service) from release assets with your `values.yaml` and by using `helm`: 
+
+    ```console
+    helm install jmeter-service https://github.com/keptn/keptn/releases/download/0.8.0/jmeter-service-0.8.0.tgz -n keptn-exec --create-namespace --values=values.yaml
+    ```
+
+* Test connection to Keptn control plane using: 
+
+    ```console
+    helm test jmeter-service -n keptn-exec
+    ```
+
+    ```console
+    Pod jmeter-service-test-api-connection pending
+    Pod jmeter-service-test-api-connection succeeded
+    NAME: jmeter-service
+    LAST DEPLOYED: Thu Feb 25 15:55:24 2021
+    NAMESPACE: keptn-exec
+    STATUS: deployed
+    REVISION: 1
+    TEST SUITE:     jmeter-service-test-api-connection
+    Last Started:   Thu Feb 25 15:55:40 2021
+    Last Completed: Thu Feb 25 15:55:42 2021
+    Phase:          Succeeded
+    ```
+
+### Summary of values to configure execution plane service
+
+See the configuration parameters of the supported execution plane services: 
+
+  - `helm-service`: [Helm Chart values](https://github.com/keptn/keptn/blob/release-0.8.0/helm-service/chart/README.md#configuration) 
+
+  - `jmeter-service`: [Helm Chart values](https://github.com/keptn/keptn/blob/release-0.8.0/jmeter-service/chart/README.md#configuration) 
+
+The important once that are used in the above example are:
+
+| Parameter                | Description             | Default        |
+| ------------------------ | ----------------------- | -------------- |
+| `distributor.stageFilter` | Sets the stage this service belongs to | `""` |
+| `distributor.serviceFilter` | Sets the service this service belongs to | `""` |
+| `distributor.projectFilter` | Sets the project this service belongs to | `""` |
+| `remoteControlPlane.enabled` | Enables remote execution plane mode | `false` |
+| `remoteControlPlane.api.protocol` | Used protocol (http, https) | `"https"` |
+| `remoteControlPlane.api.hostname` | Hostname of the control plane cluster (and port) | `""` |
+| `remoteControlPlane.api.apiValidateTls` | Defines if the control plane certificate should be validated | `true` |
+| `remoteControlPlane.api.token` | Keptn API token | `""` |
+
```

Comment:
We should add an **uninstall** seciton aswell.

It's rather simple though:
```
helm uninstall jmeter-service -n keptn-exec
helm uninstall helm-service -n keptn-exec
```

Reply:
I added the section.